manifests/plugin/remote_execution/script.pp

```diff
@@ -29,9 +31,10 @@
 #
 # $listen_on::          Proxy feature listens on https, http, or both
 #
-class foreman_proxy::plugin::remote_execution::ssh (
+class foreman_proxy::plugin::remote_execution::script (
   Boolean $enabled = true,
   Foreman_proxy::ListenOn $listen_on = 'https',
+  Enum['ssh', 'ssh-async'] $mode = 'ssh',
```

Comment:
Maybe I'm being pedantic, but I'd prefer keeping it on the old place so the diff is as small as possible.

Reply:
Up to you, I just get annoyed hunting "important" parameters that significantly affect the behavior of the class. I can do it later.

Reply:
@ekohl The root issue that there isn't a trailing comma on the last parameter.  My preference is to always have the trailing comma to avoid the extra line in the diff.

Reply:
This is seems to be the only class in this mod that doesn't have the trailing comma, so I've opened #740.